src/core/caniuse.js

```diff
@@ -104,10 +104,8 @@ function getNormalizedConf(conf) {
   if (Array.isArray(browsers)) {
     const invalidBrowsers = browsers.filter(browser => !BROWSERS.has(browser));
     if (invalidBrowsers.length) {
-      const names = invalidBrowsers.map(b => `"\`${b}\`"`).join(", ");
-      const msg =
-        `Ignoring invalid browser(s): ${names} in ` +
-        "[`respecConfig.caniuse.browsers`](https://github.com/w3c/respec/wiki/caniuse)";
+      const names = joinAnd(invalidBrowsers.map(b => `"\`${b}\`"`));
```

Comment:
For consistently using `mapper` arg..
```suggestion
      const names = joinAnd(invalidBrowsers, b => `"\`${b}\`"`);
```

Reply:
Update: Can use `mdJoinAnd` instead.

Reply:
As a followup, I need to create a new "quoted code mapper". I'll add the suggestion for now tho.